README.md

```diff
@@ -1,4 +1,4 @@
-![](https://img.shields.io/badge/-Women%20GitHubers-blueviolet.svg)
+﻿![](https://img.shields.io/badge/-Women%20GitHubers-blueviolet.svg)
```

Comment:
Remove?

Reply:
Removed, I just opened a new request instead. :D thanks for being patient with me, still learning!